node/forcedroid.js

```diff
@@ -38,7 +38,8 @@ var path = require('path');
 var outputColors = require('../external/shared/node/outputColors');
 var commandLineUtils = require('../external/shared/node/commandLineUtils');
 var shelljs = require('shelljs') ;
-var fs = require('fs'); 
+var fs = require('fs');
```

Comment:
@helenren Yes, I would prefer to use ```fs``` only, for now. In ```5.0```, I have a plan to get rid of the post install steps and some of these copy steps.